<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResetPassword</class>
 <widget class="QDialog" name="ResetPassword">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>300</width><height>180</height></rect>
  </property>
  <property name="windowTitle">
   <string>إعادة تعيين كلمة المرور</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="new_password">
     <property name="placeholderText">
      <string>كلمة المرور الجديدة</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>     
	  </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="confirm_password">
     <property name="placeholderText">
      <string>تأكيد كلمة المرور</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>   
	  </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_submit">
     <property name="text">
      <string>تغيير</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
